debian/homestead.init.d

```diff
@@ -190,6 +190,7 @@ get_daemon_args()
                      --server-name=\"$server_name\"
                      --impu-cache-ttl=$impu_cache_ttl
                      --hss-reregistration-time=$hss_reregistration_time
+                     --reg-max-expires=$reg_max_expires
```

Comment:
Yes, I did test live - that code only has any effect if the hss_reregistration time isn't set, and I explicitly set it to 0. I've removed it anyway, as it's no longer required, and unhelpful.